internal/operator/subscription.go

```diff
@@ -97,12 +98,22 @@ func (c operatorClient) ListPackageManifests(ctx context.Context, list *pkgserve
 	}
 
 	if len(filter) > 0 {
+		var notFound []string
+		wasFound := false
 		for _, f := range filter {
 			for _, pkgm := range tmppkgmlist.Items {
 				if pkgm.Name == f {
 					list.Items = append(list.Items, pkgm)
+					wasFound = true
+					break
 				}
 			}
+			if !wasFound {
+				notFound = append(notFound, f)
+			}
+		}
+		if len(notFound) > 0 {
+			return fmt.Errorf("Package(s) %v not found in catalog source", notFound)
```

Comment:
In particular, I have no idea what the proper way to bubble this error message up to the user on the terminal would be. I tried to follow and read about the 'err' go error idiom, but figured it'd be easier to ask!

As the code is now, this message doesn't show to the user--the error handling just flows up to "Unable to list PackageManifests." which isn't what we want.

Reply:
You can add 
```return types.Error{Msg: "Unable to list PackageManifests. " + err.Error()}
```
to https://github.com/opdev/opcap/blob/2660fba5702bc954bd2a8d913f8cb23d470925b0/cmd/check.go#L54 to actually show the error

Reply:
Yes